app/scripts/templates/mainmenu.eco

```diff
@@ -105,6 +105,7 @@
         <ul>
             <li><a >Help Page</a></li>
             <li><a >Dative Documentation</a></li>
+            <li><a >Version <%= @version %></a></li>
         </ul>
```

Comment:
Yeah, that could work, although I've removed the Help menu item for now because it had no content. However, in keeping with the "try to emulate Mac OS X" approach I've been going for, I think we may want to display put this in a new menu item under "Dative" called "About Dative".